paymentsheet/src/main/java/com/stripe/android/paymentsheet/model/PaymentOption.kt

```diff
@@ -55,6 +59,11 @@ constructor(
         throw IllegalStateException("Must pass in an image loader to use iconDrawable.")
     }
 
+    @OptIn(DelicateCoroutinesApi::class)
+    private var delegateDrawableScope: CoroutineScope = GlobalScope
```

Comment:
It's a little weird to have these be `var` when they're only set in constructors. If it's not too annoying, might be worth an update.

Reply:
since the primary constructor is part of our public API (though deprecated), we can't add them to constructor without breaking binary compatibility. We did the same thing for lightThemeIconUrl, darkThemeIconUrl and imageLoader. I agree it's weird but I don't think I can actually fix this :/

Reply:
We _should_ be able to do this with a default internal/private constructor. I'm happy to take this as a follow up though if you'd like.

Reply:
that'd be great! I tried something earlier which I _think_ is what you're describing. It'd be helpful to see where I went wrong :)

Reply:
Here's the follow up: https://github.com/stripe/stripe-android/pull/8441

I'll rebase after you merge this PR.

Reply:
thank you!